src/sage/geometry/lattice_polytope.py

```diff
@@ -3025,8 +3196,7 @@ def normal_form(self, algorithm="palp", permutation=False):
         elif algorithm == "palp_modified":
             result = self._palp_modified_normal_form(permutation=permutation)
         else:
-            raise ValueError('Algorithm must be palp, ' +
-                             'palp_native, or palp_modified.')
+            raise ValueError("algorithm must be 'palp', 'palp_native', or 'palp_modified'")
```

Comment:
Can you add a doctest for this ValueError?

Reply:
Done in 0bd2a93f84e